CentralityiOS/CentralityiOS/ViewControllers/ModifyTaskModalViewController.m

```diff
@@ -24,9 +30,63 @@ - (void)viewDidLoad {
     else if([self.modifyMode isEqualToString:kAddTaskMode]){
         [self initModalForAddTaskMode];
     }
+    
+    NSMutableAttributedString *toInput = [[NSMutableAttributedString alloc] initWithString:self.taskTitleInput.text attributes:@{NSForegroundColorAttributeName: [UIColor whiteColor]}];
+    
+    self.taskTitleInput.attributedText = toInput;
+    
+    [self.taskTitleInput addTarget:self action:@selector(textFieldDidChange:) forControlEvents:UIControlEventEditingChanged];
+    self.taskTitleInput.delegate = self;
+    self.taskDescInput.delegate = self;
+    IQKeyboardManager.sharedManager.enable = true;
+}
+
+
+- (void)textFieldDidChange :(UITextField *) textField{
+    NSMutableAttributedString *toInput = [[NSMutableAttributedString alloc] initWithString:textField.text attributes:@{NSForegroundColorAttributeName: [UIColor whiteColor]}];
+    
+    textField.attributedText = toInput;
+    
+    NSArray *todayKeywords = @[@"today", @"now", @"EOD"];
+        [self highlightKeyword:todayKeywords inInputField:textField methodToExecute:@selector(todayKeywordAction)];
+    
+    NSArray *tomorrowKeywords = @[@"tmrw", @"tomorrow", @"2mrw"];
+        [self highlightKeyword:tomorrowKeywords inInputField:textField methodToExecute:@selector(tomorrowKeywordAction)];
+}
+
+-(void)todayKeywordAction{
+    NSLog(@"Today Variant Found!");
+    self.taskDueDate = NSDate.date;
+    [self reloadDueDateView:self.taskDueDate];
+}
+-(void)tomorrowKeywordAction{
+    NSLog(@"Tomorrow Variant Found!");
+    NSDate *newDate = [NSDate.date dateByAddingDays:1];
+    self.taskDueDate = newDate;
+    [self reloadDueDateView:self.taskDueDate];
 }
 
--(void)initModalForEditTaskMode{
+- (void)highlightKeyword : (NSArray*) keywords inInputField:(UITextField*) inputField methodToExecute:(SEL)methodToExecute{
```

Comment:
while objective-c lets you have unspecified `NSArray*`, it's usually preferable to have the type specified - `NSArray<NSString *> *` in this case - to prevent weird bugs.